ext/relationships/main.php

```diff
@@ -217,4 +213,36 @@ private function set_has_children(int $parent_id): void
             ["has_children" => $children > 0, "pid" => $parent_id]
         );
     }
+
+    public static function has_siblings(int $image_id): bool
+    {
+        global $database;
+
+        $image = Image::by_id($image_id);
+
+        $count = $database->get_one(
+            "SELECT COUNT(*) FROM images WHERE id!=:id AND parent_id=:pid",
+            ["id" => $image_id, "pid" => $image['parent_id']]
+        );
+
+        if ($count > 0) {
+            return true;
+        }
+        return false;
+    }
+
+    public static function get_siblings(int $image_id): array
```

Comment:
To indicate that this function returns "an array of Images", rather than "an array of anything"
```suggestion
    /**
     * @return Image[]
     */
    public static function get_siblings(int $image_id): array
```

Reply:
fixed